src/atsphinx/typst/writer.py

```
@@ -170,6 +170,84 @@ def _escape(txt: str) -> str:
+                builder.config.mermaid_cmd = str(confdir_cmd)
+
+        # Temporarily set imagedir using builder's _images_dir so render_mm creates files there
+        original_imagedir = getattr(builder, 'imagedir', None)
+        original_imgpath = getattr(builder, 'imgpath', None)
+        images_dir_name = builder._images_dir.name
+        builder.imagedir = images_dir_name
+        builder.imgpath = images_dir_name
+
+        try:
+            relfn, outfn = render_mm(self, code, mermaid_options, _fmt='svg', prefix='mermaid')
+        except:
```

Ruff (0.15.18)

[error] Do not use bare `except`

(E722)